### PR TITLE
Add post item to chat from the actor sheet

### PR DESCRIPTION
**Pull request by coderunner**
Add post to chat from actor sheet.
Works for critical injuries, talents, gears, weapons, explosives, armors.

Not too sure about some translations for Spanish, German.

Screen Shot 2022-03-11 at 3 42 38 PM

### CODE REVIEW

File: css/yzecoriolis.css [hunk 1]
```diff
@@ -1751,6 +1751,9 @@
   color: white;
   transform: scale(1.05);
 }
+.yzecoriolis.sheet.actor .sheet-body .critical-injuries-list .crit-control-widget.item-delete {
+  padding-left: 15px;
```
Comment (coderunner) [inline css/yzecoriolis.css:1755]: Added a bit of padding-left to the delete button to help reduce accidental deletion of items.

File: css/yzecoriolis.css [hunk 2]
```diff
@@ -98,14 +98,14 @@
   font-style: normal;
   font-weight: 400;
   font-display: swap;
-  src: url(https://fonts.gstatic.com/s/martel/v5/PN_xRfK9oXHga0XtYQ.ttf) format('truetype');
+  src: url(https://fonts.gstatic.com/s/martel/v9/PN_xRfK9oXHga0XtYQ.ttf) format('truetype');
```
Comment (coderunner) [inline css/yzecoriolis.css:101]: This change comes from the new response from Google Font API at https://fonts.googleapis.com/css2?family=Martel:wght@400;800&family=Roboto:wght@300;400;500&display=swap

**Review by mkalus — COMMENTED** [review]
German translations look ok for me.

**Comment by AlexOkafor** [comment]
@coderunner This is great! Thanks for the contribution!

**Pull request merged**